### PR TITLE
Improve getting started Coder CTA

### PR DESCRIPTION
**Pull request by kylecarbs**
This wasn't very standout-ish before, and I think it's wise for us to experiment with directing users to Coder.

image

### CODE REVIEW

File: patches/getting-started.diff [hunk 1]
```diff
@@ -19,32 +19,89 @@ Index: code-server/lib/vscode/src/vs/workbench/contrib/welcomeGettingStarted/bro
...
+-				reset(leftColumn, startList.getDomElement(), recentList.getDomElement());
+-				reset(rightColumn, gettingStartedList.getDomElement());
++				reset(leftColumn, startList.getDomElement(), recentList.getDomElement(), gettingStartedList.getDomElement());
++				reset(rightColumn, gettingStartedCoder);
+ 				recentList.setLimit(5);
+ 			}
+ 			else {
+ 				this.container.classList.add('noWalkthroughs');
+-				reset(leftColumn, startList.getDomElement());
++				reset(leftColumn, startList.getDomElement(), gettingStartedCoder);
```
Comment (jsjoeio) [inline patches/getting-started.diff:101]: Should we gate these column changes behind
```typescript
if (this.contextService.contextMatchesRules(IsEnabledCoderGettingStarted))
```
so they don't affect users who disable this?

Comment (kylecarbs) [reply patches/getting-started.diff:101]: Good point!

Comment (kylecarbs) [reply patches/getting-started.diff]: Fixed!

**Review by jsjoeio — APPROVED** [review]
Looks way better!

**Pull request merged**